packages/edit-site/src/components/sidebar-navigation-screen-patterns/index.js

```diff
@@ -62,7 +62,7 @@ function ThemePatternsGroup( { categories, currentCategory, currentType } ) {
 	return (
 		<>
 			<div className="edit-site-sidebar-navigation-screen-patterns__group-header">
```

Comment:
This was removed in https://github.com/WordPress/gutenberg/pull/52570, so no need to worry about the title here.